changelog/algorithm_mutation_swapat.dd

```diff
@@ -0,0 +1,4 @@
+Title: `algorithm.mutation.swapAt` was exposed
+Description:
+$(XREF algorithm, mutation, swapAt) allows to swap elements
```

Comment:
Oh thanks - but unfortunately it seems like these converted files won't be merged anyhow before the next release